samples/bluetooth/bap_broadcast_sink/src/main.c

```diff
@@ -1641,7 +1642,7 @@ int main(void)
 
 		sync_bitfield = bis_index_bitfield & requested_bis_sync;
 		stream_count = 0;
-		for (int i = 1; i < BT_ISO_MAX_GROUP_ISO_COUNT; i++) {
+		for (int i = 0; i < BT_ISO_MAX_GROUP_ISO_COUNT; i++) {
```

Comment:
This is another bugfix not related to the subgroup change

Reply:
Danm. Then I will put it in the "Fix BIS index mask variable" commit